frontend/src/metabase/query_builder/components/view/ViewHeader/ViewTitleHeader.module.css

```diff
@@ -1,13 +1,57 @@
 .FilterButton {
-  transition: background 300ms linear, border 300ms linear;
+  transition: background 300ms linear;
 
-  &:hover:not([data-css-specifity-hack="🤣"]) {
+  &:hover:not([data-css-specificity-hack="🤣"]) {
```

Comment:
The way we use mantine, it's injecting all the styles into the <head> tag. This will change in mantine v7, but for now we have to deal with it. I don't like using `!important` because it's a line-item specificity hack that's very difficult to reason about. I'll look into layers though, that might be a good way forward. 👍

Reply:
I've looked into mantine layers, and
1) there is no mention of them at all in the mantine v6 docs
2) the mantine v7 docs specify that it needs to be set up globally, which we certainly have not done. We should keep this in mind for the mantine v7 conversion
3) I think my specificity hack is far superior to `!important` because it is easily removable when we upgrade to mantine v7

Reply:
Yup, let's use the hack!